apps/src/blockly/addons/functionEditor.js

```diff
@@ -179,10 +179,21 @@ export default class FunctionEditor {
     const type = procedureType || existingProcedureBlock.type;
     const isBehavior = type === BLOCK_TYPES.behaviorDefinition;
 
+    const workspace = this.editorWorkspace;
+    // Used to create and render an SVG frame instance.
+    const getBlockColor = function () {
```

Comment:
Can we turn this into a helper function somewhere so it can be reused here and in `procedureBlocks` and `behaviorBlocks`? We could have `isBehavior` be a parameter so the function is reusable.

Reply:
Updated.